lib/plugins/application.js

```diff
@@ -0,0 +1,147 @@
+var util = require('util');
+var assert = require('assert');
+var semver = require('semver');
+var format = require('util').format;
+var PluginBase = require('../plugin-base');
+
+module.exports = function(options) {
+  return new Application(options);
+};
+
+function Application(options) {
+  PluginBase.call(this, options, 'application', 'config');
+}
+
+util.inherits(Application, PluginBase);
+
+function patchAppLoopback(app) {
+  if (app.loopback) return;
+  // app.loopback was introduced in 1.9.0
+  // patch the app object to make loopback-boot work with older versions too
```

Comment:
I think it's ok to drop support for LB 1.x apps in the new version of loopback-boot and remove `patchAppLoopback` completely.